assets/airports/lszh.json

```diff
@@ -0,0 +1,1424 @@
+            ]
+        },
+        "VEBIT": {
+            "icao": "VEBIT",
+            "name": "Vebit",
+            "suffix": {"10":"2E", "16":"3S", "28":"3W", "32":"2N", "34":"2H"},
+            "rwy": {
+                "10": ["^_RWY10KLO02.1DME", ["_KLO360002.7", "A40+"], ["BREGO", "A50+"]],
+                "16": [["^_RWY16KLO01DME", "S210-"], ["_KLO360001.5", "S210-|A40+"], "_WIL053027", ["^BREGO", "A50+"]],
+                "28": ["^_RWY28KLO03DME", "_BREGO053004", ["^BREGO", "A50+"]],
+                "32": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+"], "_TRA190011", ["BREGO", "A50+"]],
+                "34": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+"], "_TRA190011", ["BREGO", "A50+"]]
+            },
+            "body": [["ZH554", "A60+"], ["ZH558", "A70+"]],
+            "exitPoints": {
+                "VEBIT": ["VEBIT"]
+            },
+            "draw": [
+                ["BREGO", "ZH554", "ZH558", "VEBIT*"]
+            ]
+        },
+        "WIL": {
+            "icao": "WIL",
+            "name": "Willisau",
+            "suffix": {"10":"2D", "16":"3R", "28":"3V", "32":"2M", "34":"2G"},
+            "rwy": {
+                "10": [["^_RWY10KLO02.1DME", "S210-"], ["_KLO360002.7", "A40+|S210-"], ["^BREGO", "A50+"]],
+                "16": [["^_RWY16KLO01DME", "S210-"], ["_KLO360001.5", "S210-|A40+"], "_WIL053027", ["BREGO", "A50+"]],
+                "28": ["^_RWY28KLO03DME", "_BREGO053004", ["BREGO", "A50+"]],
+                "32": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+"], "_TRA190011", ["BREGO", "A50+"]],
+                "34": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+"], "_TRA190011", ["BREGO", "A50+"]]
+            },
+            "body": [["ZH555", "A60+"], ["ZH551", "A70+"]],
+            "exitPoints": {
+                "WIL": [["WIL", "A70+"]]
+            },
+            "draw": [
+                ["BREGO", "ZH555", "ZH551", "WIL*"]
+            ]
+        },
+        "ZUE": {
+            "icao": "ZUE",
+            "name": "Zurich East",
+            "suffix": {"10":"1D", "16":"2R", "28":"2V", "32":"3L", "34":"3F"},
+            "rwy": {
+                "10": [["^_RWY10KLO02.1DME", "S210-"], ["_ZUE232007", "S210-"]],
+                "16": [["^_RWY16KLO01DME", "S210-"], ["_ZUE232007", "S210-"]],
+                "28": [["^_RWY28KLO03DME", "S210-"], ["_KLO253005", "S210-"], ["^ZH552", "S210-"], "_KLO232005", "KLO"],
+                "32": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["KLO", "A50+"]],
+                "34": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["KLO", "A50+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        }
+        
+    },
+    "stars": {

```

Comment:
A couple of notes before I review the STAR's in detail:
- Most of the restrictions are not as shown in the charts
- Are the RNAV transition for runways 28 and 34 used often? If so, it would make sense to add them to the runway section of the STAR's.

Reply:
- I added some restrictions due to realism, f.e. for NEGRA1A, in the real world they are about FL190 at NEGRA, without the restriction the aircrafts would be descending too low already for an rwy 14 approach.
- No not really. But if you want to, I'll add them.

Reply:
Even if that's the case, we want to follow the real restrictions as closely as possible. Regarding the transitions, it doesn't make sense to add them unless they're used very often. Since they can only be used when ATC clears aircraft for them, we can leave them out.